src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-11-01 #}
+{# wf_published_on: 2017-11-01 #}
+{# wf_blink_components: Blink>Network,Blink>Loader #}
+
+<!--
+  Aspect ratio CSS, Copyright 2017 Google Inc
+  Maintains aspect ratio in blocks that use the class, so that content doesn't
+  move around as media loads.
+
+  Adapted from https://github.com/sgomes/css-aspect-ratio
+-->
+<style>
+.aspect-ratio {
+  /* aspect-ratio custom properties */
+  /* The width portion of the aspect ratio, e.g. 16 in 16:9. */
+  --aspect-ratio-w: 1;
+  /* The height portion of the aspect ratio, e.g. 9 in 16:9. */
+  --aspect-ratio-h: 1;
+
+  position: relative;
+  max-width: 100%;
+  margin-bottom: 1ex;
+}
+
+.aspect-ratio > *:first-child {
+  width: 100%;
+}
+
+@supports (--custom-props: "true") {
+  .aspect-ratio::before {
+    display: block;
+    padding-top: calc(var(--aspect-ratio-h, 1) /
+        var(--aspect-ratio-w, 1) * 100%);
+    content: "";
+  }
+
+  .aspect-ratio > *:first-child {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+  }
+}
+</style>
+
+# Resource Prioritization – Getting the Browser to Help You {: .page-title }
+
+{% include "web/_shared/contributors/sgomes.html" %}
+
+Not every byte that is sent down the wire to the browser has the same degree of
+importance, and the browser knows this. Browsers have heuristics that attempt to
+make a best-guess at the most important resources to load first — such
+as CSS before scripts and images.
+
+That said, as with any heuristic, it doesn’t always work out; the browser might
+make the wrong decision, usually because it doesn’t have enough information at
+that time. This article explains the various tools you have at your disposal to
+influence the priority of content adequately in modern browsers, by letting them
+know what you’ll be needing later.
```

Comment:
This is a bit of a run-on sentence. And while tools are how you do it, I think the most important focus should be on the developer's end goal. How about something like this:

This article explains how to influence the priority of content adequately in modern browsers by letting them know what you'll need later.

Reply:
Done.